scripts/migrations/migrate_2.3.0.js

```diff
@@ -0,0 +1,106 @@
+const shell = require("shelljs");
+const { readContracts } = require("../util/utils.js");
+const hre = require("hardhat");
+const ethers = hre.ethers;
+const network = hre.network.name;
+// const { getStateModifyingFunctionsHashes } = require("../../scripts/util/diamond-utils.js");
+const tag = "HEAD";
+const version = "2.3.0";
+
+const config = {
+  // status at 451dc3d. ToDo: update this to the latest commit
+  addOrUpgrade: [
+    "DisputeResolverHandlerFacet",
+    "FundsHandlerFacet",
+    "MetaTransactionsHandlerFacet",
+    "OfferHandlerFacet",
+    "OrchestrationHandlerFacet1",
+    "ProtocolInitializationHandlerFacet",
+    "SellerHandlerFacet",
+    "TwinHandlerFacet",
+  ],
+  remove: [],
+  skipSelectors: {},
+  facetsToInit: {},
+  initializationData: "0x",
+};
+
+async function migrate(env) {
+  console.log(`Migration ${tag} started`);
+  try {
+    console.log("Removing any local changes before upgrading");
+    shell.exec(`git reset @{u}`);
+    const statusOutput = shell.exec("git status -s -uno scripts");
+
+    if (statusOutput.stdout) {
+      throw new Error("Local changes found. Please stash them before upgrading");
+    }
+
+    if (env != "upgrade-test") {
+      console.log("Installing dependencies");
+      shell.exec(`npm install`);
+    }
```

Comment:
```suggestion
    if (env != "upgrade-test") {
      console.log("Installing dependencies");
      shell.exec(`npm install`);
    }
```

I believe we should run even when env is "upgrade-test", otherwise the dry-run result can be different than when running for real since OZ could import could change their contracts which affects our contracts

Reply:
(which is the case for v2.3)

Reply:
Changed.